### DISCUSSION TITLE
Unable to connect Server to domain using a nginx proxy

### DISCUSSION THREAD

**Discussion by Elekam** [itzg/docker-minecraft-server#1960] [Q&A]
Hello, my server is already playable under local Server-IP:25565.
I now created the subdomain minecraft.domain.com, and would like to bind my server to that subdomain. 
I have a docker container with Nginx Proxy manager, which routes all my servers services to my subdomains.
So when accessing minecraft.subdomain.com, you reach my Nginx Docker Container listening on my servers port 80/443, and the container tries to redirect you to my servers port 25565.
I am not sure how to do this properly. 

I saw the "PROXY: Proxy:port option, and tried different IPs but none seemed to work (Nginx Docker IP, my servers IP, etc.)

When I try to go to the subdomain with my browser I get this file:
[Error-file.zip](https://github.com/itzg/docker-minecraft-server/files/10714054/Error-file.zip)
I realize that accessing it through my browser wont work, I just wanted to note that I do receive a response.

What do I do? What IP to use for the Proxy variable? My Nginx Docker Containers IP?
What to set in my Nginx Proxy Manager? I assume no SSL and just plain http routing?

Please let me know what I might be doing wrong or whether this is even possible to do through a proxy?

Thanks in advance!

**Comment by itzg** [comment]
You can't use nginx as a Minecraft protocol router unless there's an extension I don't know about. Minecraft uses a proprietary TCP protocol and not HTTP(S). 

You need to use a minecraft-aware proxy/router such as bungeecord or https://github.com/itzg/mc-router.
